doc/providers/form.rst

```diff
@@ -132,15 +132,21 @@ form by adding constraints on the fields::
         'translator.domains' => array(),
     ));
 
-    $form = $app['form.factory']->createBuilder('form')
```

Comment:
The updated example does not work for older versions of Symfony. I think we need to keep both examples and tell the user which version to use depending on the version of Symfony they have.